mora/service/address.py

```diff
@@ -415,7 +416,7 @@ def edit(self, id, req):
 
         if not old_entry:
             raise exceptions.ValidationError(
-                'original required!',
+                Error.V4
```

Comment:
This is an example of how the code is _less_ readable now. Previously, you'd know what the error was from this location alone — now you have to inspect the value in `errors.py`.